foghorn-edge/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/*
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.